great_expectations/rule_based_profiler/profiler.py

```diff
@@ -223,7 +248,7 @@ def profile(
         if include_citation:
             expectation_suite.add_citation(
                 comment="Suite created by Rule-Based Profiler with the configuration included.",
-                profiler_config=self._profiler_config,
+                profiler_config=self._citation,
```

Comment:
@cdkini Is this correct? profiler config being set to citation?..

Reply:
resolving per our conversation